<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLabellingMainWindow</class>
 <widget class="QMainWindow" name="QLabellingMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QLabelling</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc/RessourcesQLabelling.qrc">
    <normaloff>:/QLabelling/QLabellingIcon.png</normaloff>:/QLabelling/QLabellingIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="_tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveLabelsImage"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="_toolbar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSaveLabelsImage"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../rc/RessourcesQLabelling.qrc">
     <normaloff>:/QLabelling/folder-open.png</normaloff>:/QLabelling/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open image to label</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../rc/RessourcesQLabelling.qrc">
     <normaloff>:/QLabelling/process-stop.png</normaloff>:/QLabelling/process-stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../rc/RessourcesQLabelling.qrc">
     <normaloff>:/QLabelling/dialog-information.png</normaloff>:/QLabelling/dialog-information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSaveLabelsImage">
   <property name="icon">
    <iconset resource="../rc/RessourcesQLabelling.qrc">
     <normaloff>:/QLabelling/document-save.png</normaloff>:/QLabelling/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save labels image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../rc/RessourcesQLabelling.qrc"/>
 </resources>
 <connections/>
</ui>
